releases/2.13.11.md

```diff
@@ -51,6 +51,15 @@ The following changes are highlights of this release:
 
 For the complete 2.13.11 change lists, see [all merged PRs](https://github.com/scala/scala/pulls?q=is%3Amerged%20milestone%3A2.13.11) and [all closed bugs](https://github.com/scala/bug/issues?utf8=%E2%9C%93&q=is%3Aclosed+milestone%3A2.13.11).
 
+## Known issues
+
+A few regressions have already cropped up:
```

Comment:
"A few" -> "A murder of"
"cropped up" -> "come surging over the levee" or today's metaphor is surging down the Dnipro

Reply:
Must. Resist. Urge. To. Wax. Poetic.